util/Logging.py

```diff
@@ -5,13 +5,22 @@
 import re
```

Comment:
purely refactoring changes in this file - the type hints were inaccurate so I updated them (I tested several limited commands and they still work fine)

Reply:
Type hinting changes are fine, would prefer to leave the actual command logic the same as it was. (F.ex. the decorator isn't broken if no ctx.author is found, someone probably just decorated a non discord command func)